File: ControleFutebolWeb/Models/Api/AuthDtos.cs
namespace ControleFutebolWeb.Models.Api
{
    public class LoginRequest
    {
        public string Username { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
    }

    public class LoginResponse
    {
        public string Token { get; set; } = string.Empty;
        public DateTime ExpiresAtUtc { get; set; }
        public string UserName { get; set; } = string.Empty;
        public string Nome { get; set; } = string.Empty;
        public bool IsAdmin { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/comentarista/futebol/data/remote/dto/AuthDtos.kt
package com.comentarista.futebol.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class LoginRequest(
    val username: String,
    val password: String
)

@Serializable
data class LoginResponse(
    val token: String,
    val expiresAtUtc: String,
    val userName: String,
    val nome: String,
    val isAdmin: Boolean
)
